docs/_content/ontwikkelaars/algemeen/notificaties.md

```diff
@@ -0,0 +1,70 @@
+# Notificaties (voor dummies)
+
+Een component moet een bericht kunnen sturen die andere componenten kunnen ontvangen zodat zij hierop kunnen acteren indien dit een interesant bericht is voor het betreffende ontvangende component.
+
```

Comment:
(kan ook op een andere plek in het document)

## Notificaties bij andere standaarden
Zowel bij Regie- en Zaakservices als bij DigiLevering is notificaties opgelost met een publish/subscribe mechanisme. Een centrale notificatiecomponent ontvangt gebeurtenissen en distribueert deze naar abonnees. Abonnees hebben de mogelijkheid zich te abonneren op bepaalde gebeurtenissen. Bij DigiLevering zijn gebeurtenissen gedefinieerd in termen van objecttypen en attributen en kunnen bovendien filters worden gedefinieerd op attributen van de objecten.

Bij Regie- en Zaakservices kunnen alleen notificaties op zaakniveau worden gestuurd, d.w.z. als een zaak wordt gecreeerd of gewijzigd. De volgende gegevens moeten worden opgenomen in het notificatie-bericht :
* Zaakidentificatie
* Zaaktype
* Zaakstatus (optioneel)

Reply:
Toegevoegd onderaan!